patch/apply_party_patches.txt

```diff
@@ -23,24 +23,21 @@ ApplyPartyPatches:
 	call GetGameDataPointer
 	seek #result
 	readbyte #temp
-	and #temp, 1
-	jumpz #temp, .daycare_lady
-	call .daycare_mon
-.daycare_lady
+	shiftleft #temp, 31 ; nonzero if bit 0 is set
+	callnz #temp, .daycare_mon
 	set #1, 7
 	call GetGameDataOffsetConstant
 	set #1, #result
 	call GetGameDataPointer
 	seek #result
 	readbyte #temp
-	and #temp, 1
-	jumpz #temp, .box_mons
+	shiftleft #temp, 31
 	; there are two extra bytes for the daycare lady we need to skip
 	; (#1 should end up on sBreedmon2, not sBreedmon2Item)
-	add #result, 2
+	readhalfword #result
+	pos #result
 	seek #result
```

Comment:
I'm not too sure of what you're trying to do here, but `pos` and `seek` are counterparts, so this isn't doing anything at all.